MC-42708: Scheduled Export does not triggered by cron (#1418)
magento/merchdocs: src/system/data-schedule-export.md

    The new scheduled export job is added to the list on the _Scheduled Import/Export_ page. From this page it can be run immediately, for testing, and edited.

{:.bs-callout-info}

> Scheduled export makes changes to Magento system configuration. After saving, make sure you follow the cache invalidation notice that will appear on top of the Magento admin page and flush the cache in order to apply the new or updated schedule.

Reviewer: ```suggestion
Scheduled export makes changes to the system configuration. After saving, make sure you address the cache invalidation notice that appears at the top of the Admin page and flush the cache in order to apply the new or updated schedule.
```
